src/internal/provider/provider.go

```diff
@@ -42,21 +46,23 @@ type Provider struct {
 	Github       github.Client
 }
 
+// RepositoryName returns the name of the repository that the provider is assumed to be living in.
 func (p Provider) RepositoryName() string {
 	return fmt.Sprintf("terraform-provider-%s", p.ProviderName)
 }
 
-// TODO custom repository url?
+// RepositoryURL returns the URL of the repository that the provider is assumed to be living in.
 func (p Provider) RepositoryURL() string {
 	return fmt.Sprintf("https://github.com/%s/%s", p.EffectiveNamespace(), p.RepositoryName())
 }
 
-func (p Provider) getRssUrl() string {
+// RSSUrl returns the URL of the RSS feed for the repository's releases.
+func (p Provider) RSSUrl() string {
```

Comment:
If we are capitalizing RSS, why nor URL?

Reply:
we should, I'll do this now.